contracts/DAITokenWrapper.sol

```diff
@@ -21,7 +21,7 @@ contract DAITokenWrapper is ITokenWrapper, Ownable, ReentrancyGuard {
     notZeroAddress(_daiTokenAddress)
     {
         daiTokenAddress = _daiTokenAddress;
-        
+        emit LogTokenAddressChanged(daiTokenAddress, owner());
     }
```

Comment:
Please change to` _daiTokenAddress` in the emitted log so as to avoid an unnecessary SLOAD